glide.lock

```diff
@@ -38,11 +40,11 @@ imports:
 - name: github.com/Sirupsen/logrus
   version: ba1b36c82c5e05c4f912a88eab0dcd91a171688f
 - name: github.com/spf13/cobra
-  version: 4cdb38c072b86bf795d2c81de50784d9fdd6eb77
+  version: ca57f0f5dba473a8a58765d16d7e811fb8027add
```

Comment:
Because you dont have a version specified in the glide.yaml when I did `glide get github.com/kelseyhightower/envconfig` it updated the lock file and pulled in a new version. 

You can reproduce this easily on master by just running the above glide get command and diffing the .lock file.